lib/Transport.js

```diff
@@ -39,11 +39,12 @@ Transport._defaultError = function(response) {
 Transport.prototype._action = function(method, url, data, callback) {
   // console.log(method, url, JSON.stringify(data, null, 2));
   var someRequest = request[method](url);
+  someRequest.set("Content-Type", "application/vnd.api+json");
   this._attachAuthToRequest(someRequest);
   if (method === "get") {
     someRequest = someRequest.query(Perry.stringify(data));
   } else {
-    someRequest = someRequest.send(data);
+    someRequest = someRequest.send(data || {});
```

Comment:
@theninj4 to always have valid JSON I now just pass an empty object if `data` is not defined. Okay for you?